framework/db/Command.php

```diff
@@ -137,19 +137,43 @@ public function getSql()
     }
 
     /**
-     * Specifies the SQL statement to be executed.
-     * The previous SQL execution (if any) will be cancelled, and [[params]] will be cleared as well.
+     * Specifies the SQL statement to be executed. The SQL statement will be quoted using [[Connection::quoteSql()]].
+     * The previous SQL execution (if any) will be cancelled, and [[params]] will be cleared as well. See [[reset()]]
```

Comment:
`The previous SQL execution (if any) will be cancelled` → `The previous SQL (if any) will be discarded`